data-structures-stacks/is-empty.js

```diff
@@ -0,0 +1,5 @@
+/* exported isEmpty */
+
+function isEmpty(stack) {
+  return (stack.print() === 'Stack { <empty> }');
+}
```

Comment:
Please try this without using the `print` method. `print` is only for debugging.